js/core/css/var.css

```diff
@@ -3,13 +3,28 @@ var:hover {
   cursor: pointer;
 }
 
-var.respec-active {
-  background: var(--respec-background-color);
-  color: var(--respec-color);
+var.respec-hl-yellow {
+  background: #f6d51f;
```

Comment:
We might need to set `color:` too for these, because we can get into this unfortunate situation: 

![screenshot 2018-04-23 14 30 31](https://user-images.githubusercontent.com/870154/39107088-378105f4-4703-11e8-92a0-91d29346b306.png)

Reply:
oh dear!

Need advice on color. If I use black, it look less visible to me but Chrome shows it as AAA accessible. If I pick white, it looks better to me but contrast ratio is weak.
Should I try complement colors?

Reply:
> Should I try complement colors?

Might be worth a try.

Reply:
complementary colors didn't turn out well. I'll look into better color schemes later, but for now, black text color.